ui/src/reducers/index.js

```diff
@@ -38,7 +38,10 @@ const initialState = {
   isAppsRequestFailed: false,
   currentUser: 'currentUser',
   isUserDropdownOpen: false,
-  isNavigationModalOpen: false,
+  navigationModal: {
+    isNavigationModalOpen: false,
```

Comment:
Since this is a property of `navigationModal`, it could be reduced to just `isOpen`, as it is implied that the prop is referring to the navigation modal.

`navigationModal.isOpen` vs `navigationModal.isNavigationModalOpen`.

Reply:
Good point and addressed now.